docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
For this reason, I currently use this header to set the filter:
`X-Spam-Report: Spam detection software, running on the system "example.org",`
the `X-Spam-Report`-tag in combination with `example.org` is unique for my mail server. I am not happy with that at all.

Why don't I use the `X-Spam-Flag`? So far, I only filter mail above the sa_refuse-score (like the `ditch_spam`-router only moves mails above this limit in the spam folder). For training purposes, I have set up my mail account:
sa > sa_refuse -> `Spam`
sa_refuse > sa > sa_tag -> `suspicious`
sa < sa_tag -> `inbox`

Not sure, how we should handle this for all users. In my case, everything above sa_refuse is nearly 100% spam. From time to time, there are some newsletters, registration mails in my `suspicious`-folder, but >90% is spam. But two spam-folders are too much for most users. If everybody uses the same `sa_refuse`-score, I can refuse this mails straight away.